src/Features/CSharp/IntroduceVariable/CSharpIntroduceVariableService.cs

```diff
@@ -103,6 +103,11 @@ protected override bool CanIntroduceVariableFor(ExpressionSyntax expression)
                 return false;
             }
 
+            if (expression is LiteralExpressionSyntax && ((LiteralExpressionSyntax)expression).IsKind(SyntaxKind.NullLiteralExpression))
+            {
+                return false;
+            }
+
```

Comment:
doh, i trusted what the bug report said: *VB does not offer this refactoring for 'Nothing'* . Will check on VB.